### DISCUSSION TITLE
Improve documentation within Directus UI

### DISCUSSION THREAD

**Discussion by jonlambert** [directus/directus#16353] [Feature Requests]
Hi there!

Firstly to reiterate a comment I've made a lot elsewhere: Directus is awesome 🥳 I've loved working with it so far.

One thing that I've found as a newcomer to Directus is that the descriptions on fields within the CMS are sorely lacking 😞 I find myself constantly switching between docs and Directus, Googling field names and what they do. The Directus docs are great -- but it would be great if the information was there in-line with the fields.

For example: 

<img width="858" alt="Screenshot 2022-11-08 at 11 06 45" src="https://user-images.githubusercontent.com/856225/200548880-e304a67a-8653-41f5-9693-7deda4126d4f.png">

As a newcomer, it's really difficult to remember what I'm configuring here. Where is "Display" rendered when compared to "Interface"?

<img width="636" alt="Screenshot 2022-11-08 at 11 07 55" src="https://user-images.githubusercontent.com/856225/200549105-163488bd-1544-4080-8607-8f228fd408bc.png">

What does this do? ^

<img width="634" alt="Screenshot 2022-11-08 at 11 08 11" src="https://user-images.githubusercontent.com/856225/200549262-7c160bce-c8f5-49b6-b031-cc315282863d.png">

^ Where am I displaying a collection...? What sort of collection? I can't seem to select a collection, but I can choose to show it's icon...

<img width="862" alt="Screenshot 2022-11-08 at 11 10 09" src="https://user-images.githubusercontent.com/856225/200549515-bee09aef-2e8b-4bd4-b171-10766b4d3a33.png">

What are "Conditions"? What are the effects of adding a constraint here?

---

Not specifically asking these questions, more just highlighting the confusion for new users when configuring Directus 🙂 The documentation is really good, but it would be good to have more of it in-line, and/or links to the relevant section of documentation.

**Comment by br41nslug** [comment]
@erondpowell

**Comment by erondpowell** [comment]
Hey there @jonlambert

Yes! updating the Data model docs is a big priority.

However, atm we have a few big projects ahead of it:
- https://github.com/directus/docs/pull/197
- Some API documentation improvements
- Improvements to the Flows docs
Then we will get to 
- https://github.com/directus/docs/issues/83

If you have any thoughts, ideas, opinions, feel free to share on the relevant issues or PRs.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
